src/modules/dashboard/components/ColonyHome/ColonyMembers/MembersSubsection.tsx

```diff
@@ -68,7 +71,7 @@ interface Props {
   maxAvatars?: number;
   members: Maybe<string[]>;
   bannedMembers?: BannedMember[];
-  isContributors: boolean;
+  isContributorsSubsection: boolean;
```

Comment:
It is a much better variable name - Iw as going to suggest something to that extent.